src/styles/Calculator.css

```diff
@@ -0,0 +1,25 @@
+.calculator-body {
+  display: grid;
+  grid-template: repeat(6, 1fr) / repeat(4, 1fr);
+}
+
```

Comment:
- [ ] Kindly reduce the width of the calculator to match the width of the given design
Your design
![Capture](https://user-images.githubusercontent.com/85095702/188749813-fa71649f-65d3-4977-bb59-31ee0785a3d7.PNG)
Given template
![figma](https://user-images.githubusercontent.com/85095702/188749839-5dc63025-5c90-4a74-b140-2daec34dd5e3.PNG)